src/hooks/api/actions/useApiTokensApi/useApiTokensApi.ts

```diff
@@ -1,11 +1,11 @@
 import useAPI from '../useApi/useApi';
 
-export interface IApiTokenCreate {
+export type IApiTokenCreate = {
```

Comment:
We usually use `interface` for objects, though I suppose the end result is the same.

Reply:
I changed it back in last update :) Good catch